src/pages/uxp/reference-js/Global Members/HTML Elements/HTMLWebViewElement.md

```diff
@@ -0,0 +1,711 @@
+
+
+<a name="htmlelement-innertext" id="htmlelement-innertext"></a>
+
+## innerText : `string`
+
+
+<a name="element-nodename" id="element-nodename"></a>
+
+## nodeName : `string`
+**Read only**
+
+
+<a name="element-localname" id="element-localname"></a>
+
+## localName : `string`
+**Read only**
+
+
+<a name="element-tagname" id="element-tagname"></a>
+
+## tagName : `string`
+**Read only**
+
+
+<a name="element-nodetype" id="element-nodetype"></a>
+
+## nodeType : `number`
+**Read only**
+
+
+<a name="element-namespaceuri" id="element-namespaceuri"></a>
+
+## namespaceURI : `string`
+**Read only**
+
+
+<a name="element-id" id="element-id"></a>
+
+## id : `string`
+
+
+<a name="element-tabindex" id="element-tabindex"></a>
+
+## tabIndex : `number`
+
+
+<a name="element-classname" id="element-classname"></a>
+
+## className : `string`
+
+
+<a name="element-attributes" id="element-attributes"></a>
+
+## attributes : `NamedNodeMap`
+**Read only**
+
+
+<a name="element-style" id="element-style"></a>
+
+## style : `Style`
+**Read only**
+
+
+<a name="element-clientleft" id="element-clientleft"></a>
+
+## clientLeft : `number`
+**Read only**
+
+
+<a name="element-clienttop" id="element-clienttop"></a>
+
+## clientTop : `number`
+**Read only**
+
+
+<a name="element-clientwidth" id="element-clientwidth"></a>
+
+## clientWidth : `number`
+**Read only**
+
+
+<a name="element-clientheight" id="element-clientheight"></a>
+
+## clientHeight : `number`
+**Read only**
+
+
+<a name="element-height" id="element-height"></a>
+
+## height : `string` \| `Number`
+The height of the element
+
+
+
+<a name="element-width" id="element-width"></a>
+
+## width : `string` \| `Number`
+The width of the element
+
+
+
+<a name="element-offsetparent" id="element-offsetparent"></a>
+
+## offsetParent : `Element`
+**Read only**
+
+
+<a name="element-offsetleft" id="element-offsetleft"></a>
+
+## offsetLeft : `number`
+**Read only**
+
+
+<a name="element-offsettop" id="element-offsettop"></a>
+
+## offsetTop : `number`
+**Read only**
+
+
+<a name="element-offsetwidth" id="element-offsetwidth"></a>
+
+## offsetWidth : `number`
+**Read only**
+
+
+<a name="element-offsetheight" id="element-offsetheight"></a>
+
+## offsetHeight : `number`
+**Read only**
+
+
+<a name="element-scrollleft" id="element-scrollleft"></a>
+
+## scrollLeft : `number`
+
+
+<a name="element-scrolltop" id="element-scrolltop"></a>
+
+## scrollTop : `number`
+
+
+<a name="element-scrollwidth" id="element-scrollwidth"></a>
+
+## scrollWidth : `number`
+**Read only**
+
+
+<a name="element-scrollheight" id="element-scrollheight"></a>
+
+## scrollHeight : `number`
+**Read only**
+
+
+<a name="element-autofocus" id="element-autofocus"></a>
+
+## autofocus : `boolean`
+Indicates if the element will focus automatically when it is loaded
+
+
+
+<a name="element-uxpcontainer" id="element-uxpcontainer"></a>
+
+## uxpContainer : `number`
+**Read only**
+
+
+<a name="element-disabled" id="element-disabled"></a>
```

Comment:
Since UXP-specific, without a description, exposing this to 3P doesn't make a lot of sense. Don't get me wrong: I think if it makes sense we should expose this to 3P, but exposing it without any information doesn't add a lot of value.

Reply:
Tracked in #77